pkgs/by-name/hp/hpipm/package.nix

```diff
@@ -26,7 +26,8 @@ stdenv.mkDerivation (finalAttrs: {
   ];
 
   cmakeFlags = [
-    "-DBLASFEO_PATH=${blasfeo}"
+    "-DHPIPM_FIND_BLASFEO=ON"
```

Comment:
Is this flag equivalent to the previous `"-DBLASFEO_PATH=${blasfeo}"` flag? Even if so, it would be nice if this change could be in a separate commit, or explained in the commit message.